### ISSUE TITLE
Add namespace aliases to class suggestions

### ISSUE THREAD

**Issue by nevadascout** [HvyIndustries/crane#229]
When suggesting classes, we should also suggest any namespace aliases that have been created.

```php
use MyNamespace\MyClass as SomethingElse;

//...

$foo = new 
//         ^ suggest "SomethingElse" here
```

**Comment by TheColorRed** [comment]
If a namespace is defined in the document, but the selected class is not defined using `use`, the class should be automatically imported (enabled/disabled via setting).

```php
namespace Path\To\Space;

// Automatically added if setting is on
use A\B\C;

$foo = new C;
```

Otherwise

```php
namespace Path\To\Space;

// Automatically Added if setting is off
$foo = new \A\B\C;
```

**Comment by nevadascout** [comment]
This would be great, but it's not something we can easily do with the completion provider.

Instead, we can register a code actions provider to notice that you have hard-coded a namespace path when referencing a class, and offer to "use" it instead.

I think this is a fair compromise, what do you think?

**Comment by nevadascout** [comment]
This is now implemented in v0.3.1
